README.pod

```diff
@@ -2,7 +2,7 @@
 
 =head1 L<LaTeXML|http://dlmf.nist.gov/LaTeXML/>
 
-=for html <a href="https://travis-ci.org/brucemiller/LaTeXML"><img src="https://travis-ci.org/brucemiller/LaTeXML.svg?branch=master"></a>
+=for html <a href="https://travis-ci.org/brucemiller/LaTeXML"><img src="https://travis-ci.org/brucemiller/LaTeXML.svg?branch=master"></a><a href="https://ci.appveyor.com/project/dginev/latexml"><img src="https://ci.appveyor.com/api/projects/status/github/brucemiller/latexml?svg=true"></a>
```

Comment:
The `dginev` in the link is not a mistake, I explained why it's there in the comment. Maybe it can be replaced if Bruce enables appveyor himself, but it should be fully functional as-is.